src/main/java/co/rsk/bitcoinj/script/Script.java

```diff
@@ -681,7 +681,11 @@ public boolean isPayToScriptHash() {
      * Returns whether this script matches the format used for multisig outputs: [n] [keys...] [m] CHECKMULTISIG
      */
     public boolean isSentToMultiSig() {
```

Comment:
Do we have tests for these method? I feel like this should be well covered. Can be done in other PR

Reply:
Exactly, this effort will be done in a different task(already created).